tljh/user.py

```diff
@@ -28,9 +37,11 @@ def ensure_user(username):
         username
     ])
 
+    pm.hook.tljh_new_user_create(username=username)
```

Comment:
We should probably move this to the end, after the chmod - in case they wanna undo the effects of the chmod.

Reply:
I agree as well